source/upgrade.txt

```diff
@@ -76,6 +76,16 @@ Version 6.x Breaking Changes
 - Automatic Encryption methods are now in the {+driver-short+}. You must
   import these methods from the driver instead of from
   ``mongodb-client-encryption``.
+- Removes support for strings with a length of 12 from the ``ObjectId``
+  constructor.
+- The ``abortTransaction`` and ``commitTransaction`` methods now return
+  ``null`` instead of the raw command results.
+- Connection strings and the ``MongoClient`` constructor now accept only
+  the boolean values ``true`` and ``false``. Other values such as ``1``, or ``"yes"``
+  are no longer valid.
+- The ``Binary`` BSON type no longer accepts a string as a constructor argument.
```

Comment:
Suggestion:

I think the related items should be closer together. For this item, the constructors for the Binary BSON type has changed. The change is that there's no longer a constructor for it that accepts strings.

To keep Binary BSON type and constructor together, you could reorder the sentence to something like this:
"Removed the Binary BSON type constructor that accepted a string."